dotnet-desktop-guide/net/winforms/data/how-to-synchronize-multiple-controls.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "How to ensure multiple controls bound to the same data source remain synchronized"
+description: Learn how to ensure multiple controls bound to the same data source remain synchronized.
+ms.date: "06/06/2022"
+dev_langs: 
+  - "csharp"
+  - "vb"
+helpviewer_keywords: 
+  - "controls [Windows Forms], binding multiple"
+  - "controls [Windows Forms], synchronizing with data source"
+ms.custom: devdivchpfy22
+---
+
+# How to ensure multiple controls bound to the same data source remain synchronized
```

Comment:
The title is too long and is missing ` (Windows Forms .NET)`

Reply:
ok. Added